backend/contributions/rubric_review.py

```
@@ -0,0 +1,181 @@
+        if value not in seen:
+            normalized.append(value)
+            seen.add(value)
+
+    if invalid:
+        raise serializers.ValidationError({
+            field_name: f"Unknown value(s): {', '.join(map(str, invalid))}."
+        })
+    return normalized
+
+
+def _normalize_score(value, section_key):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_normalize_score`

(ANN202)